health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/SelectingBoundariesDuplicate.js

```
@@ -21,19 +21,31 @@ const SelectingBoundariesDuplicate = ({ onSelect, formData, ...props }) => {
   );
   const [executionCount, setExecutionCount] = useState(0);
   const [showPopUp, setShowPopUp] = useState(null);
-  const [updateBoundary , setUpdateBoundary] = useState(true);
+  const [updateBoundary, setUpdateBoundary] = useState(true);
+  const [currentStep, setCurrentStep] = useState(2);
+  const currentKey = searchParams.get("key");
   const [key, setKey] = useState(() => {
     const keyParam = searchParams.get("key");
     return keyParam ? parseInt(keyParam) : 1;
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)